src/shared/components/features-section/index.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import FeatureItem from 'shared/components/features-section/feature-item';
+import Feat1Svg from '-!svg-react-loader!shared/media/images/feat1.svg';
+import Feat2Svg from '-!svg-react-loader!shared/media/images/feat2.svg';
+import Feat3Svg from '-!svg-react-loader!shared/media/images/feat3.svg';
+import Feat4Svg from '-!svg-react-loader!shared/media/images/feat4.svg';
+import Feat5Svg from '-!svg-react-loader!shared/media/images/feat5.svg';
+
+import styles from './index.module.css';
+
+const Features = ({ intl }) => {
+    const featsArr = [
```

Comment:
What do you think of having this static data in another file? The component would be much cleaner, without this array and all the imports.

Reply:
Fixed.